libraries/AP_Airspeed/AP_Airspeed_MSP.cpp

```diff
@@ -50,6 +50,11 @@ void AP_Airspeed_MSP::handle_msp(const MSP::msp_airspeed_data_message_t &pkt)
         press_count /= 2;
     }
 
+    if (pkt.temp == INT16_MIN) {
```

Comment:
This looks good if the airspeed sensor on the node has never sent any temperature update.
is it actually possible that a sensor still send the pressure but and invalid temperature?
in that case we will continue using the old stale temperature in the FC

Reply:
once a sensor sends a temperature once then it will always send one with the pressure